### PR TITLE
Hotfixes Spells Evoker

### PR DESCRIPTION
**Pull request by ipro4911**
This is a WIP or in progess v1

**Changes proposed:**
Adding 2 new spells Deep Breath and Landslide
-  
-  
-  

**Issues addressed:**
When first casted it doesnt do anything just do animation standing still

Closes #  (insert issue tracker number)
Missing Empowered mechanic of spell

**Tests performed:**

(Does it build, tested in-game, etc.)
Didnt test it yet i will update this after i do

**Known issues and TODO list:** (add/remove lines as needed)

- [One day if anyone knows can be added mechanic] 
- [ ] 

I will add after i see if its worth the sql hotfixes

### CODE REVIEW

File: CMakeCache.txt
Comment (grip244) [inline CMakeCache.txt:1]: I don't think you should be adding Cmake to your PR.

File: src/server/scripts/Spells/spell_evoker.cpp [hunk 1]
```diff
@@ -203,6 +205,99 @@ class spell_evo_pyre : public SpellScript
...
+            Unit* caster = GetCaster();
+
+            if (!caster->IsFalling())
+                return SPELL_FAILED_NOT_ON_GROUND;
+
+            return SPELL_CAST_OK;
+
+
+        }
+         //TODO: need id of SPELL_EVOKER_DEEP_BREATH_VISUAL,SPELL_EVOKER_DEEP_BREATH_EMPOWERED_BUFF  and empowered damage mechanic too
```
Comment (grip244) [inline src/server/scripts/Spells/spell_evoker.cpp:253]: https://www.wowhead.com/spell=353759/deep-breath#triggered-by (buff)
https://www.wowhead.com/spell=362010/deep-breath The Jump Charge.

File: CMakeCache.txt
Comment (ipro4911) [reply CMakeCache.txt:1]: i forgot to remove it so i had to close try to delete pull request Sorry

File: src/server/scripts/Spells/spell_evoker.cpp (hunk 1 above)
Comment (ipro4911) [reply src/server/scripts/Spells/spell_evoker.cpp:253]: wao ok didnt know it had another id i will change it soon or add what missing

**Comment by ipro4911** [comment]
Any news? cuz on some pservers the script got fixed added stuff i think i was reading in theese days something with game object or smth https://github.com/TrinityCore/TrinityCore/pull/21395 Dummy Script

**Comment by grip244** [comment]
> Any news? cuz on some pservers the script got fixed added stuff i think i was reading in theese days something with game object or smth #21395 Dummy Script

 This is supposed to be your pr, no one is gonna do your homework. Ask for help I'm discord, you gotta do it yourself man.

**Pull request closed without merging**